MetaMask/metamask-mobile
# fix: Remove wallet connect and sdk connect prefix from ppom domain

## **Description**

Domains from wallet connect - are being passed to ppom with a wc:: preffix (example: wc::https://furrevertoken.com/). This is causing url parsing to fail inside PPOM and needs to be fixed on MM side.

## **Related issues**

Fixes: [#2221](https://github.com/MetaMask/MetaMask-planning/issues/2221)

## **Manual testing steps**

1. Connect to testdapp from real device
2. Perform a malicious transaction
3. Click Report an issue
4. Expand the details drop down
5. See that domain is not prefixed with wc::

## **Screenshots/Recordings**

### **Before**
![IMG_0563](https://github.com/MetaMask/metamask-mobile/assets/44811/de438794-244a-4133-b8af-e913d02e044b)

### **After**
![IMG_0564](https://github.com/MetaMask/metamask-mobile/assets/44811/96acbce6-89a9-40e0-a372-0feb1110ac2a)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
